guardian/support-frontend
# One Time Checkout - Payment Method selection

## What are you doing in this PR?

Creating the Payment method selection section of the one time checkout form. Re-using and refactoring styles shared with the generic checkout.



[**Trello Card**](https://trello.com/c/skCq3Wwk/1029-one-time-checkout-payment-selection)

## Why are you doing this?

Ongoing work creating a new one time checkout following the same standards as our generic checkout.



## How to test

Go to `us/one-time-checkout?product=SupporterPlus` and see the new payment selection section. Selecting card brings up the stripe form with recaptcha.

Generic checkout should look exactly as it did before.




## Screenshots
![image](https://github.com/user-attachments/assets/98227aea-12dc-4e5e-bff1-df39cc302be5)
![image](https://github.com/user-attachments/assets/d08ad1f1-044e-4f89-a687-8a6fb2e73f6b)
